tests/Unit/NumericalAlgorithms/SphericalHarmonics/Test_Strahlkorper.cpp

```diff
@@ -86,18 +84,25 @@ void test_phys_spec_constructor_consistency() {
 
   std::uniform_real_distribution<double> distribution(0.0, 1.0);
   MAKE_GENERATOR(generator);
+
+  // create Strahlkorper using radius at collocation points
   const auto radius = make_with_random_values<DataVector>(
       make_not_null(&generator), distribution,
       DataVector(physical_size, std::numeric_limits<double>::signaling_NaN()));
-
-  const Strahlkorper<Frame::Inertial> s_physical(
-      l_max, l_max, radius, center,
-      ylm::StrahlkorperConstructorData::RadiusAtCollocationPoints);
-  const Strahlkorper<Frame::Inertial> s_spectral(
-      l_max, l_max, s_physical.coefficients(), center,
-      ylm::StrahlkorperConstructorData::SpectralCoefficients);
-
-  CHECK(s_physical == s_spectral);
+  const Strahlkorper<Frame::Inertial> strahlkorper_physical(l_max, l_max,
+                                                            radius, center);
+
+  // create Strahlkorper using spectral coefficients
+  const size_t spectral_size = ylm::Spherepack::spectral_size(l_max, l_max);
+  ModalVector spectral_coefficients(
+      spectral_size, std::numeric_limits<double>::signaling_NaN());
+  std::copy(strahlkorper_physical.coefficients().begin(),
```

Comment:
[optional] You can avoid this extra copy by making `spectral_coefficients` non-owning

```cpp
const DataVector& coefs = strahlkorper_physical.coefficients();
const ModalVector spectral_coefficients{coefs.data(), coefs.size()};
const Strahlkorper<Frame::Inertial> strahlkorper_spectral(
      l_max, l_max, spectral_coefficients, center);
```

Reply:
I unfortunately was not able to get this to compile so I left it as-is